app/invocation/invocation_action_card.tsx

```diff
@@ -0,0 +1,165 @@
+import pako from "pako";
+import React from "react";
+import format from "../format/format";
+import InvocationModel from "./invocation_model";
+import { build } from "../../proto/remote_execution_ts_proto";
+import rpcService from "../service/rpc_service";
+
+interface Props {
+  model: InvocationModel;
+  search: string;
+}
+
+interface State {
+  contents?: ArrayBuffer;
+  action_array?: Uint8Array;
+  action?: build.bazel.remote.execution.v2.Action;
+  command?: build.bazel.remote.execution.v2.Command;
+  error?: string;
+}
+
+export default class ActionCardComponent extends React.Component<Props, State> {
+  state: State = {};
+  componentDidMount() {
+    this.fetchAction();
+  }
+
+  fetchAction() {
+    // TODO: Replace localhost:1987 with the remote cache address and prefix the path with the instance name
+    let actionFile = "bytestream://localhost:1987/blobs/" + this.props.search.substring(14);
+    rpcService
+      .fetchBytestreamFile(actionFile, this.props.model.getId(), "arraybuffer")
+      .then((action_buff: any) => {
+        let temp_array = new Uint8Array(action_buff);
```

Comment:
`temp_array` doesn't feel like a super descriptive name - I'd either go with `actionArray` or get rid of the variable an inline this below in the setState call